### DISCUSSION TITLE
IPv6 WAN 状态一直显示未连接，实际已分配ipv6地址

### DISCUSSION THREAD

**Discussion by sarabear** [coolsnowwolf/lede#9517] [R68S / 电犀牛专区]
IPv6 WAN 状态一直显示未连接，实际已分配ipv6地址。 
之前的openwrt不管了删了WAN6或者WAN6设置为DHCPv6，都能正确获取v6 ip和显示， 现在的WAN6无法设置DHCPv6（没了DHCPv6协议了，选DHCP的话实际会获取v4的ip）， 删除后首页还是显示  IPv6 WAN未连接。  不过实际IPV6已经可以用，设备也能正常访问v6的网站，就是看着不舒服。
![IMG_20220611_184511](https://user-images.githubusercontent.com/1264982/173184570-1cd8aa56-e1de-4375-95e5-c1e691d60779.jpg)

**Comment by wabcwell** [comment]
我也是同样的问题
还有就是路由器本身有IPV6地址么？

**Reply by ljlwdf** [reply]
IPv6 WAN 状态显示无的问题，安装luci-proto-ipv6，就会显示正常。

**Comment by Sn0r1axi** [comment]
期待修复，有相同问
![~P$8BUD}OB5P623)8C%~_ZN](https://user-images.githubusercontent.com/26668216/173213002-ccfbaa0a-39fd-4e10-bf17-f7753561e98f.png)
题

**Comment by zhang73331** [comment]
相同情况，IPV6测试通不过
